modules/tinymce/src/core/main/ts/delete/TableDelete.ts

```diff
@@ -66,8 +70,24 @@ const setCursorInCell = (editor: Editor, cell: SugarElement<HTMLTableCellElement
  * - part of a table and content outside is selected
  */
 const emptySingleTableCells = (editor: Editor, cells: SugarElement<HTMLTableCellElement>[], outsideDetails: Optional<OutsideTableDetails>): boolean => {
-  // Remove content from selected cells
-  cleanCells(cells);
+  const rng = editor.selection.getRng();
+  const startInTable = SelectorExists.ancestor(SugarElement.fromDom(rng.startContainer), 'table');
```

Comment:
This shouldn't be done... we should be using the `isStartInTable` property from `outsideDetails`.

Reply:
Nice I missed that being available